### DISCUSSION TITLE
getaddrinfo ENOTFOUND localhost

### DISCUSSION THREAD

**Discussion by elarbor** [drawdb-io/drawdb#69] [General]
项目启动不了
<img width="1003" alt="image" src="https://github.com/drawdb-io/drawdb/assets/43592730/357260ef-a3f9-4b40-b63f-111b71addcab">

**Comment by 1ilit** [comment]
Looks like it can't find the url for localhost. Do you have it configured in /etc/hosts? 

There should be an entry in /etc/hosts for localhost: 
```console
127.0.0.1       localhost
```
